docs/user-guide/gaffer-basics/what-is-aggregation.md

```diff
@@ -0,0 +1,70 @@
+# What is Aggregation?
```

Comment:
It might be good to explain why we do aggregation or why this is a feature and how it is useful. Potentially with a reference/link to aggregation with sketches as this is a big benefit of it but not obvious.

Reply:
Good point